docs/adrs/0017-laptop-security.md

```diff
@@ -0,0 +1,22 @@
+# ADR-0017 Laptop Security
+
+## Context
+
+When people leave their laptop open and unlocked this could be a big security breach, especially when you have SSO on
```

Comment:
```suggestion
When people leave their laptop open, unlocked, and unattended, this could be a big security breach, especially when you have SSO on
```